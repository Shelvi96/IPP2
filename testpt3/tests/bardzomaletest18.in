NEW ID

9:475>4
154>0
364690;63>:91253
654232:73>06
:>66876;51
3>39879245
78:33>7169304
;22580>77:
67>;56:
85>24:
2>0:
04:9025>97247470
55:35>73
18>30282
8488242:;:>30203958
91:9;221:4>:99:355
432;84;;>4;;5017
@9551:427290;2834731:67656057
@9387895000794:
@4583891247;2;52:6
@6558::
@251:352929932402365084;52
@;:39:560419::3807
@4;475:826
@;8;:;6;1055873:96:9515451:
@141032321:562;02:30;931746
@1;9101249883
