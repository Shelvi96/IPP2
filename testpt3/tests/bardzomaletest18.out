32321681090150400
17
16225
0
6305387904398
30745
0
72495142505698
72780668535190
0
